integration_tests/e2e/suspensionOfVisits.cy.ts

```diff
@@ -0,0 +1,60 @@
+import SuspensionOfVisitsPage from '../pages/suspensionOfVisits'
+import Page from '../pages/page'
+
+context('Suspensions', () => {
+  const orderId = '1234567'
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.signIn()
+    cy.visit(`/orders/${orderId}/suspension-of-visits`)
+  })
+
+  it('is reachable', () => {
+    Page.verifyOnPage(SuspensionOfVisitsPage)
+  })
+
+  describe('timeline component', () => {
+    it('Renders a suspensions timeline', () => {
+      const suspensions = Page.verifyOnPage(SuspensionOfVisitsPage)
+      suspensions.timeline().should('be.visible')
+    })
+  })
+
+  describe('Suspensions timeline item', () => {
+    it('Includes expected column headers', () => {
+      const suspensions = Page.verifyOnPage(SuspensionOfVisitsPage)
+
+      suspensions.timelineItems().each($item => {
+        cy.wrap($item).within(() => {
+          suspensions.details().within(() => {
+            suspensions.detailsSummary().click({ force: true })
```

Comment:
{force:true} added here and elsewhere to avoid cypress error: 'is not visible because it has an effective width and height of: 25 x 0 pixels'